modules/ob-release-notes-1-1-1.adoc

```diff
@@ -0,0 +1,15 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-1-1_{context}"]
+= Release notes for {builds-shortname} General Availability 1.1.1
+
+{builds-shortname} General Availability (GA) 1.1.1 is now available on {ocp-product-title} 4.16.
+
+[id="fixed-issues-1-1-1_{context}"]
+== Fixed issues
+
+The following section highlights fixed issues in {builds-shortname} 1.1.1.
+
+* With this update, the Shared Resource Container Storage Interface (CSI) Driver has permissions to create subject access reviews and get/list/watch Shared Resource objects.
```

Comment:
```suggestion
* With this update, the Shared Resource Container Storage Interface (CSI) Driver has the following permissions:
* Create subject access reviews
* Get, list, or watch Shared Resource objects
```

This is an optional suggestion depending upon whether `Get, list, and watch` mean the same thing.